ml_tools/eolearn/tests/test_sampling.py

```diff
@@ -14,10 +14,65 @@
 import pytest
 from numpy.testing import assert_array_equal
 from pytest import approx
+from shapely.geometry import Point, Polygon
 
 from eolearn.core import EOPatch, FeatureType
 from eolearn.ml_tools import BlockSamplingTask, FractionSamplingTask, GridSamplingTask, sample_by_values
-from eolearn.ml_tools.sampling import expand_to_grids
+from eolearn.ml_tools.sampling import expand_to_grids, random_point_in_triangle
+
+
+@pytest.mark.parametrize(
+    "triangle, generator, values",
+    [
+        (
+            Polygon([[-10, -12], [5, 10], [15, 4]]),
+            np.random.default_rng(seed=42),
```

Comment:
i guess you could just put a `seed` here? :thinking:
maybe it doesn't even have to be parametrized according to the seed (since you're using the same in all three cases, gets rid of three lines)

Reply:
updated